File: Programacion/Python/Calculadora Simple v3/calculadora.py
#Suma

def suma(a,b):
   return a + b

#Resta

def resta(a,b):
    return a - b

#Multiplicación

def multiplicación(a,b):
    return a * b

#División

def división(a,b):
    if a != 0 and b != 0:
        return a / b
    else:
        print("No es posible dividir por 0")
    
if __name__ == "__main__":
    v1 = int(input("Dame el primer valor: "))
    v2 = int(input("Dame el segundo valor: "))
    print(suma(v1,v2))
    print(resta(v1,v2))
    print(multiplicación(v1,v2))
    print(división(v1,v2))
    

File: Programacion/Python/.vscode/calculadora.c
#include <stdio.h>

int main() {
    int Operacion = 0;
    int Valor1, Valor2;
    float Resultado;

    // Pregunta al usuario qué tipo de operación quiere hacer
    do {
        printf("Elige una opción: 1. Sumar 2. Restar 3. Multiplicar 4. Dividir 5. Salir: ");
        scanf("%d", &Operacion);

        switch (Operacion) {
            case 1: // Suma
                printf("Dame el primer valor: ");
                scanf("%d", &Valor1);
                printf("Dame el segundo valor: ");
                scanf("%d", &Valor2);
                Resultado = Valor1 + Valor2;
                printf("Tu resultado es: %.2f\n", Resultado);
                break;

            case 2: // Resta
                printf("Dame el primer valor: ");
                scanf("%d", &Valor1);
                printf("Dame el segundo valor: ");
                scanf("%d", &Valor2);
                Resultado = Valor1 - Valor2;
                printf("Tu resultado es: %.2f\n", Resultado);
                break;

            case 3: // Multiplicación
                printf("Dame el primer valor: ");
                scanf("%d", &Valor1);
                printf("Dame el segundo valor: ");
                scanf("%d", &Valor2);
                Resultado = Valor1 * Valor2;
                printf("Tu resultado es: %.2f\n", Resultado);
                break;

            case 4: // División
                printf("Dame el primer valor: ");
                scanf("%d", &Valor1);
                printf("Dame el segundo valor: ");
                scanf("%d", &Valor2);
                if (Valor2 != 0) {
                    Resultado = (float)Valor1 / Valor2;
                    printf("Tu resultado es: %.2f\n", Resultado);
                } else {
                    printf("Error: No se puede dividir entre cero.\n");
                }
                break;

            case 5: // Salir
                printf("Saliendo del programa.\n");
                break;

            default:
                printf("Opción no válida. Inténtalo de nuevo.\n");
                break;
        }
    } while (Operacion != 5);

    return 0;
}
